source/clear-linux/get-started/virtual-machine-install/vmw-player-preconf.rst

```diff
@@ -39,25 +39,25 @@ Install the VMware Workstation Player hypervisor
    Download your preferred version.
 
 #. Depending on which OS you're running, install it by following one of these
-    instructions:
+   instructions:
 
    * On supported Linux distros:
 
-      #. Enable a GUI desktop.
+     #. Enable a GUI desktop.
 
-      #. Start a terminal emulator.
+     #. Start a terminal emulator.
 
-      #. Start the installer by issuing the command below and follow the guided
-         steps.
+     #. Start the installer by issuing the command below and following the
+        guided steps.
 
-         .. code-block:: console
+        .. code-block:: console
```

Comment:
This should code-block should be "bash".

Reply:
I changed this and one other instance from console to bash--given this is new plan of record.

Reply:
Correction: I will leave the final instance of console as is, under "Enable UEFI boot support". Stet.